python/src/trezorlib/cli/cardano.py

```diff
@@ -84,8 +84,8 @@ def sign_tx(client, file, protocol_magic, network_id, testnet):
     )
 
     return {
-        "tx_hash": signed_transaction.tx_hash.hex(),
-        "serialized_tx": signed_transaction.serialized_tx.hex(),
+        "tx_hash": transaction_parts.tx_hash.hex(),
+        "signatures": [witness[1].hex() for witness in transaction_parts.signatures],
```

Comment:
It could make sense to somehow decode the path from `List[int]` back to `str` and show it here. But the other coins also just show signatures (not even hash)...
Let's leave this to the Trezor guys?

Reply:
The consumer of the CLI output is a human, I think that's why we already `.hex()` everything there.
This is why I'd convert the paths back if it was easy. But since I couldn't find it implemented yet, I'm fine with returning `List[int]`.